web-workout-tracker/workout/src/components/Marketing.js

```diff
@@ -17,9 +17,15 @@ const Marketing = (props) => {
   const { user } = useAuth0();
 
   React.useEffect(() => {
-    if(user !== undefined){
+    // if(document.cookie.indexOf( 'auth0.is.authenticated' ) != -1){
+    //   console.log('works')
+    // }
+    if(document.cookie.indexOf( 'auth0.is.authenticated' ) == -1){
```

Comment:
Maybe === will be better in this case?

Reply:
For some reason it doesnt work with the ===, that was my original idea